src/views/AuthLayout.tsx

```diff
@@ -1,25 +1,55 @@
+import React from "react";
```

Comment:
why is the title of this file AuthLayout.tsx (how is it related to authentication?)

Reply:
AUTHORIZED LAYOUT. YOU ONLY SEE IF YOU ARE AUTHORIZED.